netbox_topology_views/views.py

```diff
@@ -300,6 +301,9 @@ def create_edge(
         if hasattr(cable, 'color') and cable.color != "":
             edge["color"] = "#" + cable.color
 
+    # if straight_cables == True: edge["smooth"] = False
+    edge["smooth"] = not straight_cables
+
```

Comment:
If this is optional (see comment above), you need to check for 'None' like in the other ones above.